library/src/main/res/layout/exomedia_exo_view_layout.xml

```diff
@@ -2,9 +2,11 @@
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
                 xmlns:tools="http://schemas.android.com/tools"
                 android:layout_width="match_parent"
-              android:layout_height="match_parent">
+              android:layout_height="match_parent"
+                android:background="@android:color/black"
```

Comment:
Hmm, that's a good point.... OK we probably should make it black (either you can do it in this PR, or I can do it later)